### PR TITLE
Release playground v2

### PR DESCRIPTION
**Pull request by toyamarinyon**
> [!NOTE]
> This pull request will be conflict with #241.

## Changes
- Move canary implementation to main playground directory
- Remove feature flag checks for playgroundV2 and viewFlag
- Reorganize beta-proto code into prev/ directory
- Update import paths across codebase
- Remove unused mock data files

### Create agent

- [ ] Visit `/agents`
- [ ] Show agents
- [ ] Create a new agent

### Move to the playground of the agent

- [ ] Visit `/agents`
- [ ] Show agents
- [ ] Click an agent
- [ ] Move to the playground of the agent clicked

### Execute a node on the playground

- [ ] Visit `/p/:agentId`
- [ ] Open playground
- [ ] Put on node
- [ ] Generate text with Text generator node

### Execute a flow on the playground
- [ ] Visit `/p/:agentId`
- [ ] Open playground
- [ ] Put on nodes
- [ ] Connect the nodes
- [ ] Execute flow with connected nodes

### CODE REVIEW

**Comment by shige** [comment]
@toyamarinyon Could you fix the deployment errors?

**Comment by toyamarinyon** [comment]
@shige oh, sorry! It seems that the #245  have to be merged.

File: app/(main)/agents/page.tsx [hunk 1]
```diff
@@ -1,55 +1,66 @@
...
 	return (
-		<div className="container mt-8">
-			<section className="text-foreground">
-				<div className="flex flex-col gap-8">
-					<div className="flex justify-between">
-						<h1>Agents</h1>
-						<CreateAgentButton createAgentAction={createAgentAction} />
+		<div className="grid gap-[16px] grid-cols-3">
+			{dbAgents.map((agent) => (
+				<Link href={`/p/${agent.id}/canary`} key={agent.id}>
```
Comment (shige) [inline app/(main)/agents/page.tsx]:
```suggestion
				<Link href={`/p/${agent.id}`} key={agent.id}>
```

Comment (shige) [reply app/(main)/agents/page.tsx]: @toyamarinyon I cannot open the agent page in `/agents`. Please check it and try your OAT.

**Comment by toyamarinyon** [comment]
@shige Sorry. I didn't check move to the playground of the agent clicked on the agent list page. I fix and check the following scenarios:


### Create agent

- [x] Visit `/agents`
- [x] Show agents
- [x] Create a new agent

### Move to the playground of the agent 🆕 

- [x] Visit `/agents`
- [x] Show agents
- [x] Click an agent
- [x] Move to the playground of the agent clicked

### Execute a node on the playground

- [x] Visit `/p/:agentId`
- [x] Open playground
- [x] Put on node
- [x] Generate text with Text generator node

### Execute a flow on the playground
- [x] Visit `/p/:agentId`
- [x] Open playground
- [x] Put on nodes
- [x] Connect the nodes
- [x] Execute flow with connected nodes

File: app/(playground)/p/[agentId]/page.tsx [hunk 2]
```diff
@@ -1,165 +1,165 @@
...
-	return {
-		...agent,
-		graphv2: {
-			...agent.graphv2,
-			xyFlow,
-			flowIndexes: [],
-		},
-	};
-}
+// This page is experimental. it requires PlaygroundV2Flag to show this page
```
Comment (shige) [inline app/(playground)/p/[agentId]/page.tsx]:
```suggestion
```

Comment (shige) [reply app/(playground)/p/[agentId]/page.tsx]: nits

**Review by shige — APPROVED** [review]
LGTM! 🎉

**Pull request merged**